lib/rubygems/security.rb

```diff
@@ -377,6 +377,11 @@ class Exception < Gem::Exception; end
   ONE_YEAR = 86400 * 365
 
   ##
+  # One day in seconds
+
+  ONE_DAY = 86400
```

Comment:
This is fine, but then I would say we could move ONE_YEAR below it and change it to:

    ONE_YEAR = ONE_DAY * 365